Project.toml

```diff
@@ -21,7 +21,8 @@ CodeTracking = "1"
 JuliaInterpreter = "0.8"
 LoweredCodeUtils = "1.2"
 OrderedCollections = "1"
-Requires = "~1.0"
+# Exclude Requires-1.1.0 - see https://github.com/JuliaPackaging/Requires.jl/issues/94
```

Comment:
Unfortunately, I think Pkg will delete this comment next time it writes the project file.

Reply:
Yes, I've noticed this. I think I understand why this is nontrivial to deal with, although establishing a convention that comments go with their next item seems like it might allow the problem to be resolved in most cases.

Reply:
Yes it will, but I still thought adding a comment was an improvement over not adding one ;-) This can either be maintained manually, or if it's deleted at least it'll appear in the git history.